sys/net/network_layer/icmpv6/icmpv6_hdr_print.c

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (C) 2017 smlng <s@mlng.net>, HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
+ * @file
```

Comment:
You were not supposed to remove the `@author` tag... Please move the author back from the copyright line to that tag.